src/handlers/gestures/gestureStateManager.ts

```diff
@@ -18,51 +18,47 @@ const warningMessage = tagMessage(
 const REANIMATED_AVAILABLE = Reanimated?.useSharedValue !== undefined;
 const setGestureState = Reanimated?.setGestureState;
 
+// ui runtime global
+declare const global: {
+  _setGestureStateNew: (handlerTag: number, state: State) => void;
+};
+
+const wrappedSetGestureState = (handlerTag: number, state: State) => {
+  'worklet';
+
+  if (REANIMATED_AVAILABLE) {
+    // When Reanimated is available, setGestureState should be defined
+    if (global._setGestureStateNew) {
```

Comment:
Let's use `globalThis` instead of `global`, I believe @tjzel can provide some more insights on that.
```suggestion
    if (globalThis._setGestureState) {
```
Also, I'm not sure if we need to include "new" in the name, how about using `_setGestureState` instead and renaming `setGestureState` to setGestureStateOld` as suggested in another comment?

Reply:
I don't think it supports React Native 0.79 yet?

```
In file included from /Users/jakubpiasecki/Projects/gesture-handler/FabricExample/node_modules/react-native-reanimated/Common/cpp/reanimated/NativeModules/ReanimatedModuleProxy.cpp:1:
In file included from /Users/jakubpiasecki/Projects/gesture-handler/FabricExample/ios/Pods/Headers/Public/React-jsi/jsi/jsi.h:13:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/functional:552:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/__functional/boyer_moore_searcher.h:25:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/array:507:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/algorithm:1846:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/__algorithm/inplace_merge.h:18:
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/__algorithm/rotate.h:14:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator18.4.sdk/usr/include/c++/v1/__algorithm/move_backward.h:44:19: error: overload resolution selected deleted operator '='
```

Reply:
I'd prefer not to change the previous method - it's consumed by RNGH, but it's defined entirely on the Reanimated part. Changing it will very likely break the integration between Reanimated and older versions of RNGH.

And `_setGestureStateNew` is subject to change, though `_setGestureState` will be hard to beat 😅